Gaming The Syllabus/Assets/Scripts/Combat/CombatUnit.cs

```diff
@@ -80,6 +83,11 @@ public void ReadXml(XmlReader reader)
         MaximumHealth = int.Parse(reader.GetAttribute("maximumHealth"));
         CurrentHealth = int.Parse(reader.GetAttribute("currentHealth"));
         dialogueColor = reader.GetAttribute("dialogueColor");
+        if (reader.GetAttribute("sprite") != null)
+        {
+            // Don't forget we don't use file extensions when loading from the resources folder
+            Sprite = Resources.Load<Sprite>(Path.Combine(Files.SpritesFolder, reader.GetAttribute("sprite")));
+        }
```

Comment:
So we will be using xml files as well for loading/storing sprites? If so, great call. It's good to be consistent and XML has treated us well.